index.html

```diff
@@ -1 +1,275 @@
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+      <link rel="preload" href="https://d33wubrfki0l68.cloudfront.net/a978f759fa0230c1e590d1bdb5a1c03ceb538cec/fed6b/fonts/elenawebregular/elenawebregular.woff2" as="font" type="font/woff2" crossorigin="">
+      <link rel="preload" href="https://d33wubrfki0l68.cloudfront.net/b324ee03d5048d2d1831100e323b0b6336ffce68/0445e/fonts/mijaregular/mija_regular-webfont.woff2" as="font" type="font/woff2" crossorigin="">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.12/css/all.css">
+      <link rel="stylesheet" href="css/main.css">
+
+  </head>
+  <body>
+    <!-- HEADER -->
+      <nav>
+        <div id="tnw-logo">
+          <a href="#">tnw</a>
+        </div>
+        <ul id='nav-tnw'>
+          <li>News</li>
+          <li>Events</li>
+          <li>Business</li>
+          <li>AMAs</li>
+          <li>Spaces</li>
+        </ul>
+        <div id="sections-container">
+          <ul id="nav-tnw-sections">
+            <li><a href="#">latest</a></li>
+            <li><a href="#">hard fork</a></li>
+            <li><a href="#">plugged</a></li>
+            <li><a href="#">the next police</a></li>
+            <li><a href="#">full stack</a></li>
+          </ul>
+          <ul id="nav-tnw-icons">
+            <li> <a href="#"><i class="fas fa-search" aria-hidden="true"></i></a> </li>
+            <li> <a href="#"><i class="fas fa-bars" aria-hidden="true"></i></a> </li>
+          </ul>
+          <ul id="nav-tnw-follows">
+            <li> <a href="#"><i class="fab fa-facebook" aria-hidden="true"></i></a> </li>
+            <li> <a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a> </li>
+            <li> <a href="#"><i class="fas fa-envelope" aria-hidden="true"></i></a> </li>
+            <li> <a href="#"><i class="fas fa-ellipsis-h" aria-hidden="true"></i></a> </li>
+          </ul>
+          <div class="nav-message">
+            <a href="#">3000+ businesses will be at TNW2019! Make yours one of them →</a>
+          </div>
+        </div>
+      </nav>
+
+    <section>
+        <!-- SECTION - (2-col) -->
+        <div id="main">
+          <section class="grid-3">
+            <article class="cover-story">
+              <div class="cover-image"> </div>
+              <a href="#">
+                <div class="cover-logo"></div>
+                <div id="cover-time">Friday — March 15, 2019</div>
+                <header class="cover-header">
+                  <h5 class="cover-section">game</h5>
+                  <h2 class="cover-title">
+                    Who are you siding with in the Spotify-Apple antitrust war?
+                  </h2>
+                  <ul class="cover-header-bottom">
+                    <li> <a href="#">by Abhimanyu</a> </li>
```

Comment:
`a` shouldn't be inside `a` tag

Reply:
Thanks. We solved it.

Reply:
Don't understand *images aren't responsive*